pyccel/ast/core.py

```diff
@@ -2907,8 +2942,11 @@ class ValuedArgument(Basic):
     >>> n
     n=4
     """
+    def __new__(cls, *args, **kwargs):
+        kwargs.pop('kwonly', False)
+        return Basic.__new__(cls, *args, **kwargs)
```

Comment:
I think we need to have a general discussion about which parameters we pass to this constructor.
For example, here it seems natural to only pass `cls`.

Reply:
👍